src/wix/WixToolset.Core/Compiler_Package.cs

```diff
@@ -55,6 +56,9 @@ private void ParsePackageElement(XElement node)
                     case "Codepage":
                         codepage = this.Core.GetAttributeLocalizableCodePageValue(sourceLineNumbers, attrib);
                         break;
+                    case "Comments":
```

Comment:
The `Comments` attribute should go on the `SummaryInformation` element, not the `Package` element. This is a change in WiX v4 as the old `Product` element was renamed `Package`, and the old `Package` element was renamed `SummaryInformation`.

Reply:
Sorry I got you wrong. I moved it to the `SummaryInformation` element